examples/forward/plot_frequency_interference.py

```diff
@@ -0,0 +1,178 @@
+# -*- coding: utf-8 -*-
+"""
+.. _ex-frequency-interference:
+
+Frequency Interference and the 1/f Component in Power Spectra
+=============================================================
+
+This example shows how the spatial offset of nearly phase-locked source time
+courses with lower frequency oscillations have much less destructive
+interference than those with higher frequencies leading to a 1/f power
+spectral density (PSD) distribution. This phenomena is commonly
+misattributed to the skull acting as a lowpass filter.
+
+.. contents:: This tutorial covers:
+   :local:
+   :depth: 2
+
+"""
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+#          John Mosher   <John.C.Mosher@uth.tmc.edu>
```

Comment:
have you asked John Mosher if he's okay with being added as an author here? :)

Reply:
Yes he said that was good